catalog/ui/catalog-ui-metacard-sharing-impl/src/main/java/org/codice/ddf/catalog/ui/sharing/ShareableMetacardImpl.java

```diff
@@ -76,7 +78,6 @@ public ShareableMetacardImpl(String id) {
    */
   public static boolean canShare(Metacard metacard) {
```

Comment:
✏️ You could break each of these sections into their own predicates to increase readability.